### DISCUSSION TITLE
Wait time problem

### DISCUSSION THREAD

**Discussion by Tahernejad** [LiveHelperChat/livehelperchat#1906] [General]
Hi,
I'm used older version of LHC, calculate of wait time is wronged in statistics sometimes.
Can you help me which file or method used for calculate this ?

**Comment by PeopleInside** [comment]
Older version of LHC are maybe no more supported.
Updated version keep your server more secure and updated with the latest fix on live helper chat web app.

Also, if you found the issue still occur in an updated version, steps on how to reproduce the issue will be useful.
How you know the time is wrong?
